<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreCenteredChangeValueOptionWidget</class>
 <widget class="QWidget" name="PreCenteredChangeValueOptionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>614</width>
    <height>281</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout" columnstretch="0,1">
     <item row="0" column="0">
      <widget class="QCheckBox" name="constFieldCheckBox">
       <property name="text">
        <string>Const field</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="constFieldDoubleSpinBox">
       <property name="minimum">
        <double>-99999.000000000000000</double>
       </property>
       <property name="maximum">
        <double>99999.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="preutteranceCheckBox">
       <property name="text">
        <string>Preutterance</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="preutteranceDoubleSpinBox">
       <property name="minimum">
        <double>-99999.000000000000000</double>
       </property>
       <property name="maximum">
        <double>99999.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="overlapCheckBox">
       <property name="text">
        <string>Overlap</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="overlapDoubleSpinBox">
       <property name="minimum">
        <double>-99999.000000000000000</double>
       </property>
       <property name="maximum">
        <double>99999.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QCheckBox" name="rightCheckBox">
       <property name="text">
        <string>Right</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDoubleSpinBox" name="rightDoubleSpinBox">
       <property name="minimum">
        <double>-99999.000000000000000</double>
       </property>
       <property name="maximum">
        <double>99999.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="tempoDoubleSpinBox">
       <property name="minimum">
        <double>1.000000000000000</double>
       </property>
       <property name="maximum">
        <double>99999.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>1.000000000000000</double>
       </property>
       <property name="value">
        <double>120.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="normalizeWithTempoButton">
       <property name="text">
        <string>Normalize with tempo</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>This tool will keep preutterance's &lt;b&gt;absolute&lt;/b&gt; position while changing other values relatively. In other words, it calculate new  &lt;b&gt;left&lt;/b&gt; value and change it based on the new pattern you input above. If new left value is less than 0, the entry's left and preutterance will be kept as original.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>88</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>constFieldCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>constFieldDoubleSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>14</y>
    </hint>
    <hint type="destinationlabel">
     <x>138</x>
     <y>17</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>preutteranceCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>preutteranceDoubleSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>80</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel">
     <x>176</x>
     <y>41</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>overlapCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>overlapDoubleSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>55</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>178</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rightCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>rightDoubleSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>56</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>194</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
